qcodes/tests/drivers/test_keysight_b220x.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+from pyvisa.errors import VisaIOError
+from qcodes.instrument_drivers.Keysight.keysight_b220x import KeysightB220X
+
```

Comment:
in case users have an actual instrument connected, shall they be warned that before running the test they should disconnect their samples or smth?

Reply:
Perhaps the tests should run using the simulated instrument by default. 
I am thinking just do

```
use_simulated=True
inst_addr = None
```
at the top and then connect to the simulated unless `use_simulated` is set to False and a valid address supplied

Reply:
yeah, that's much clearer.

Reply:
As per @astafan8's suggestion in https://github.com/QCoDeS/Qcodes/pull/1373#discussion_r231457289, I changed the default physical instrument address to something that will require the user who wants to run the test with the physical instrument to actively change this address in the test code. Without that the test will always execute using the simulated instrument by default.